site/content/docs/howto/cicd.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Using Pinniped for CI/CD cluster operations
+description: Using Pinniped for CI/CD cluster operations.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: Use Pinniped for CI/CD
+    weight: 500
+    parent: howtos
+---
+
+This guide shows you how to configure Pinniped and your CI/CD system of choice to use Pinniped to administrate Kubernetes clusters.
```

Comment:
```suggestion
This guide shows you how to configure Pinniped and your CI/CD system of choice to use Pinniped to authenticate into Kubernetes clusters.
```

Reply:
I like the term "administrate" in the doc since it calls out that CI/CD needs to do things within that cluster. Pinniped's job is to enable administration by providing authentication.

Reply:
But it's somewhat poorly worded here - Pinniped does not itself administrate the cluster. CI/CD should administrate the cluster.